src/slack/app_home.service.ts

```diff
@@ -377,50 +378,20 @@ export class AppHomeService {
               break;
           }
           break;
-        case 'disconnect':
-          switch (connectionInfo.type) {
-            case SUPPORTED_INTEGRATIONS.POSTGRES:
-              await this.integrationsService.removePostgresConfig(teamId);
-              break;
-            case SUPPORTED_INTEGRATIONS.HUBSPOT:
-              await this.integrationsService.removeHubspotConfig(teamId);
-              break;
-            case SUPPORTED_INTEGRATIONS.JIRA:
-              await this.integrationsService.removeJiraConfig(teamId);
-              break;
-            case SUPPORTED_INTEGRATIONS.SALESFORCE:
-              await this.integrationsService.removeSalesforceConfig(teamId);
-              break;
-            case SUPPORTED_INTEGRATIONS.GITHUB:
-              await this.integrationsService.removeGithubConfig(teamId);
-              break;
-            case SUPPORTED_INTEGRATIONS.NOTION:
-              await this.integrationsService.removeNotionConfig(teamId);
-              break;
-            case SUPPORTED_INTEGRATIONS.LINEAR:
-              await this.integrationsService.removeLinearConfig(teamId);
-              break;
-            case 'mcp':
-              await this.integrationsService.removeMcpConnection(teamId, connectionInfo.id);
-              break;
-            case SUPPORTED_INTEGRATIONS.OKTA:
-              await this.integrationsService.removeOktaConfig(teamId);
-              break;
-            default:
-              break;
-          }
-          await slackWorkspace.reload({
-            include: TOOL_CONNECTION_MODELS
-          });
-          await webClient.views.publish({
-            user_id: userId,
-            view: await this.getHomeView({
-              slackWorkspace,
-              connection: undefined,
-              userId
-            })
+        case 'disconnect': {
+          const connectionName =
+            connectionInfo.type === 'mcp'
+              ? 'MCP Server'
+              : (INTEGRATIONS.find((i) => i.value === connectionInfo.type)?.name ??
```

Comment:
Variable names should always be meaningfull, avoid using one letter or acronyms for variable names
```suggestion
              : (INTEGRATIONS.find((integration) => integration.value === connectionInfo.type)?.name ??
```

Reply:
Got it, I have made the change in the latest commit.